### ISSUE TITLE
A11y_AzureDataStudioSQL_Project_CreateNewProject_ScreenReader: NVDA not announces the path information which is the disabled content present under "Workspace" using any key

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#14316]
### Actual:
NVDA does not announces the path information which is the disabled content present under "Workspace" using any key.

Same issue is not observed with Jaws

Similar issue is observed in below scenarios:

- Open Existing Project- Project

### Expected:
NVDA should announce path information which is the disabled content present under "Workspace".

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Element:Text;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.26.0-insider
Windows Version: Win10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
4. Start NVDA
5. Navigate till "Workspace" title and use insert + numpad 4 or 6 and check nvda announcement

### User Impact:
Screen reader user will not get all the information about the disabled content.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by alanrenmsft** [comment]
I tried Narrator and it is able to get to the disabled items using items mode, this is more of a NVDA issue, I don't think there is anything we can do about it.

**Comment by Shivamd1** [comment]
The issue does not reprduces in nvda scan mode.

**Comment by fsteffi** [comment]
GithubTags:#Closed;
